### PR TITLE
fix account team project update method

### PR DESCRIPTION
**Pull request by ivan-savciuc**
`project_name` field should not be in the PUT request body

### CODE REVIEW

File: accounts_acc_test.go [hunk 1]
```diff
@@ -110,9 +110,17 @@ var _ = Describe("Accounts", func() {
 			if errTP := client.AccountTeamProjects.Create(
 				account.Account.Id,
 				team.Team.Id,
-				AccountTeamProject{ProjectName: projectName, TeamType: projectType}); errTP != nil {
+				AccountTeamProject{ProjectName: projectName, TeamType: "developer"}); errTP != nil {
 				Fail("cannot create account team project association:" + errTP.Error())
 			}
+
+			By("Update account team project")
+			if errTPu := client.AccountTeamProjects.Update(
```
Comment (nkchern-avn) [inline accounts_acc_test.go:118]: AFAIR `errTPu` is not a go-style. `err` is enough(I saw that the other code uses this which is still not so correct)

File: accounts_acc_test.go [hunk 2]
```diff
@@ -110,9 +110,17 @@ var _ = Describe("Accounts", func() {
 			if errTP := client.AccountTeamProjects.Create(
 				account.Account.Id,
 				team.Team.Id,
-				AccountTeamProject{ProjectName: projectName, TeamType: projectType}); errTP != nil {
+				AccountTeamProject{ProjectName: projectName, TeamType: "developer"}); errTP != nil {
```
Comment (nkchern-avn) [inline accounts_acc_test.go:113]: can we actually check in tests that we don't have `projectName` field in the request body? Or mb I'm missing something...

File: accounts_acc_test.go (hunk 1 above)
Comment (ivan-savciuc) [reply accounts_acc_test.go:118]: Here in the acceptance tests, we do have many errors, and I did add an explicit postfix to an error and clearly see where this error is coming from.

No "go style" manifest or linter forbids naming errors differently from `err`, so I prefer to leave it this way for readability reasons.

File: accounts_acc_test.go (hunk 2 above)
Comment (ivan-savciuc) [reply accounts_acc_test.go:113]: This is an acceptance test, and we create real resources on the Aiven side when executing it. The idea here is that we test Create and Update methods, so we are associating a project to a team with one type and then updating to another. Exactly what is the customer is doing who has this issue.

**Review by nkchern-avn — APPROVED** [review]

**Pull request merged**